### PR TITLE
features(nosqlbench): nosqlbench 5.0 for Scylla Java driver v4.x stress testing

### PR DESCRIPTION
**Pull request by CodeLieutenant**
### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: defaults/test_default.yaml [hunk 1]
```diff
@@ -208,7 +208,7 @@ enable_argus: true
 stress_image:
   ndbench: 'scylladb/hydra-loaders:ndbench-jdk8-20210720'
   ycsb: 'scylladb/hydra-loaders:ycsb-jdk8-20220918'
-  nosqlbench: 'scylladb/hydra-loaders:nosqlbench-4.15.49'
+  nosqlbench: 'codelieutenant/nosqlbench:latest'
```
Comment (CodeLieutenant) [inline defaults/test_default.yaml:211]: Will be changed after testing with CST in argus

**Comment by fruch** [comment]
@CodeLieutenant 

seem like you have an extra commit here, which isn't relevant

this is usually the flow we are working on PR/feature branches 
```bash
git fetch upstream
git rebase upstream/master
git push origin feat/nosqlbench-5.0 -f
```

File: docker/nosqlbench/build.sh [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    esac
+done
+
+
+if [[ -z "$VERSION" ]]; then
+    VERSION="5.21.2-release"
+fi
+
+if [[ -z "$SCYLLA_JAVA_DRIVER_VERSION" ]]; then
+    SCYLLA_JAVA_DRIVER_VERSION="4.18.0.1"
```
[the remark this answers is not in the document]
Comment (CodeLieutenant) [reply docker/nosqlbench/build.sh]: Build script removed, instructions left how to build Docker image in the README

**Comment by fruch** [comment]
@CodeLieutenant got to run the SCT case with this one ?

**Comment by CodeLieutenant** [comment]
> @CodeLieutenant got to run the SCT case with this one ?

Still running

**Pull request closed without merging**